mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb

```diff
@@ -1004,7 +1004,7 @@ func flattenBucketLifecycleRuleAction(action *storage.BucketLifecycleRuleAction)
 
 func flattenBucketLifecycleRuleCondition(condition *storage.BucketLifecycleRuleCondition) map[string]interface{} {
 	ruleCondition := map[string]interface{}{
-		"age":                        int(condition.Age),
+		"age":                        int(*condition.Age),
```

Comment:
It looks like this is causing the downstream provider is failing to compile:

```
/go/pkg/mod/cloud.google.com/go/storage@v1.22.1/bucket.go:1504:30: cannot use r.Condition.AgeInDays (variable of type int64) as type *int64 in struct literal
/go/pkg/mod/cloud.google.com/go/storage@v1.22.1/bucket.go:1592:30: cannot use rr.Condition.Age (variable of type *int64) as type int64 in struct literal
```

This is because resource_storage_bucket uses [google.golang.org/api/storage/v1](https://github.com/GoogleCloudPlatform/magic-modules/blob/e0c2da8e2607dd33a0384f42ad72ea537b62cbe5/mmv1/third_party/terraform/resources/resource_storage_bucket.go.erb#L23)

Please make sure the downstream provider can compile and that tests pass locally - steps 5 & 6 of [General Contributing Steps](https://github.com/GoogleCloudPlatform/magic-modules#general-contributing-steps)